guacamole-ext/src/main/resources/org/apache/guacamole/protocols/rdp.json

```diff
@@ -109,6 +109,7 @@
                         "en-gb-qwerty",
                         "en-us-qwerty",
                         "es-es-qwerty",
+                        "es-es-dvorak",
```

Comment:
This is a minor thing, but `es-es-dvorak` should be _above_ `es-es-qwerty`, alphabetically - here, and in the other files you've updated in the client code.

Reply:
Done. Now checking how to fix server's PR build.